xchange-huobi/src/main/java/org/knowm/xchange/huobi/HuobiAdapters.java

```diff
@@ -91,7 +90,7 @@ static ExchangeMetaData adaptToExchangeMetaData(
 
     Map<Currency, CurrencyMetaData> currencies = new HashMap<>();
     for (HuobiCurrencyWrapper huobiCurrencyWrapper : currencyWrapper) {
-      if (huobiCurrencyWrapper.getHuobiCurrencies().length != 0) {
+      if(huobiCurrencyWrapper.getHuobiCurrencies().length != 0) {
```

Comment:
why are you changing this?

Reply:
Just reformatting, as I see your comments on the formatting and during the formatting phase, I see this part as different in formatting and fixed it as I see. I can revert it if desired but I think it is over kill to revert it.

Reply:
This comes from the other commits which you have already merged into knownm/develop. I can revert it if desired.

Reply:
Fixed